package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "build": "turbo run build",
     "generate": "turbo run generate",
-    "dev": "env-cmd turbo run dev --parallel",
+    "dev": "env-cmd -f .env env-cmd -f .env.local turbo run dev --parallel",
```

Comment:
Do I understand this correctly?
- it loads `.env` and `.env.local` files from the root of monorepo
- each package should also load their env vars by using `packages/env-vars` in the `dev` script (for example NODE_OPTIONS=`--require env-vars`)?

Reply:
It loads the variables from the root.
Other packages should use `dotenv` or `env-cmd` or whatever to load their local variables.

Reply:
One more thing - it doesn't work if I don't use have `.env.local` file. By default it's not stored in the repository, so the script ideally should ignore that file if it doesn't exist, so it just works™️ after cloning the repo

Reply:
Good catch @witoszekdev

Reply:
Fixed